docs/src/use/configure/configuration-files.md

```diff
@@ -69,6 +69,7 @@ Each configuration object contains all of the information ESLint needs to execut
 * `name` - A name for the configuration object. This is used in error messages and config inspector to help identify which configuration object is being used. ([Naming Convention](#configuration-naming-conventions))
 * `files` - An array of glob patterns indicating the files that the configuration object should apply to. If not specified, the configuration object applies to all files matched by any other configuration object.
 * `ignores` - An array of glob patterns indicating the files that the configuration object should not apply to. If not specified, the configuration object applies to all files matched by `files`. If `ignores` is used without any other keys in the configuration object, then the patterns act as [global ignores](#globally-ignoring-files-with-ignores) and it gets applied to every configuration object.
+* `extends` - An array of strings, configuration objects, or configuration arrays that contain additional configuration to apply.
```

Comment:
Perhaps there should be a note that this property is available only when using `defineConfig()`?

Reply:
All of the documentation is updated to use `defineConfig()` with this PR, so I don't think this distinction is necessary (and it may cause more confusion...why would I ever not use `defineConfig()`?)

Reply:
Okay, makes sense.